internal/engine/build_and_deployer_live_update_test.go

```diff
@@ -0,0 +1,226 @@
+package engine
+
+import (
+	"strings"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+	"github.com/windmilleng/tilt/internal/k8s"
+	"github.com/windmilleng/tilt/internal/model"
+	"github.com/windmilleng/tilt/internal/store"
+	"github.com/windmilleng/tilt/internal/synclet/sidecar"
+)
+
+type testCase struct {
+	env k8s.Env
+
+	baseManifest        model.Manifest
+	liveUpdSyncs        []model.LiveUpdateSyncStep
```

Comment:
i'm surprised the test case doesn't take a LiveUpdate object? and then factor out this code into a helper that makes LiveUpdate objects